reward/reward_distributor.go

```diff
@@ -38,113 +50,469 @@ type governanceHelper interface {
 	ParamsAt(num uint64) (*params.GovParamSet, error)
 }
 
-type RewardDistributor struct {
-	rcc *rewardConfigCache
-	gh  governanceHelper
+type rewardConfig struct {
+	// hardfork rules
+	rules params.Rules
+
+	// values calculated from block header
+	totalFee *big.Int
+
+	// values from ChainConfig
+	mintingAmount *big.Int
+	minimumStake  *big.Int
+	deferredTxFee bool
+
+	// parsed ratio
+	cnRatio    *big.Int
+	kgfRatio   *big.Int
+	kirRatio   *big.Int
+	totalRatio *big.Int
+
+	// parsed KIP82 ratio
+	cnProposerRatio *big.Int
+	cnStakingRatio  *big.Int
+	cnTotalRatio    *big.Int
+}
+
+type RewardSpec struct {
+	Minted   *big.Int                    // the amount newly minted
+	Fee      *big.Int                    // total tx fee spent
+	Burnt    *big.Int                    // the amount burnt
```

Comment:
```suggestion
	TotalFee      *big.Int                    // total tx fee spent
	BurntBurnt    *big.Int                    // the amount burnt
```
Please change these names as per latest [KIP-82](https://github.com/yeri-lee/kips/blob/temp/KIPs/kip-82.md).

Reply:
You mean BurntFee?

Reply:
fixed in 0a0ef72c